### PR TITLE
Changeling metashield

### PR DESCRIPTION
**Pull request by BombasterDS**
## About the PR
Adds changeling in metashield rules

:cl:
- tweak: Updated Metashield rule for changeling

## Summary by CodeRabbit

- **New Features**
	- Introduced a new section on "Changelings" in the roleplay rules, clarifying their shielded nature and interaction guidelines for players.
	- Specified conditions under which players may gain knowledge about Changelings, including their abilities and behaviors.

### CODE REVIEW

**Comment by Aidenkrz** [comment]
We don't have metashield rules because theyre nigh impossible to enforce in LRP. This would fit in MRP more.

**Comment by BombasterDS** [comment]
> We don't have metashield rules because theyre nigh impossible to enforce in LRP. This would fit in MRP more.

wdym, we have wizden metashield rule in game 😱

File: Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml [hunk 1]
```diff
@@ -31,6 +31,20 @@
   - a War Ops announcement
   - being a nuclear operative
 
+  ## Changelings
+
+  Changeling in the eyes of the public are extinct creatures and no one would believe they could appear on the station is shielded.
```
Comment (deltanedas) [inline Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml]: bad wording for the second half

File: Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml [hunk 2]
```diff
@@ -31,6 +31,20 @@
...
+  This metashield prohibits players from bloodtesting.
+  
+  Despite this, many people know about changeling ability to change their appearance, impossibility of gibbing the body and stasis ability is not shielded.
```
Comment (deltanedas) [inline Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml]: bad wording for the end

File: Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml [hunk 3]
```diff
@@ -31,6 +31,20 @@
...
+
+  The revealing condition for this shield is any of the following:
+  - discovering a hollowed body
+  - witnessing changeling use their non-stealth abilities (not stings or passive abilities)
+  - witnessing someone trying to absord a body
```
Comment (deltanedas) [inline Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml]: absorb

File: Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml [hunk 4]
```diff
@@ -31,6 +31,20 @@
...
+  - witnessing someone changing it's appearance
```
Comment (deltanedas) [inline Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml]: its

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by whateverusername0 — APPROVED** [review]

**Review by Piras314 — APPROVED** [review]

File: Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml [hunk 5]
```diff
@@ -31,6 +31,20 @@
   - a War Ops announcement
   - being a nuclear operative
 
+  ## Changelings
+
+  In the eyes of the public, changeling are an exterminated predatory lifeform and their existence on the station is shielded.
+  This metashield prohibits players from crew bloodtesting.
```
Comment (Piras314) [inline Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml:37]: until the metashield is revealed

File: Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml [hunk 6]
```diff
@@ -31,6 +31,20 @@
...
+  
+  The knowledge of changelings ability to change their appearance and their possibility to resurrect from dead state is not shielded.
```
Comment (Piras314) [inline Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4Metashield.xml:39]: change to "Knowledge about the abilities of Changelings is not shielded"

**Review by Piras314 — APPROVED** [review]

**Comment by Piras314** [comment]
Admin review period will be over in two days (only posted it yesterday, sorry)

**Pull request merged**